CI is failing because the health tests don't succeed anymore.
```
79/179 Test  #538: health-tests/json-test.toit ..........................................***Failed    0.15 sec
--- /home/runner/work/toit/toit/build/host/tmp/GOLD_YoC8HwGk0NCt	2026-05-18 21:16:05.419367591 +0000
+++ /home/runner/work/toit/toit/build/host/tmp/OUTPUT_YoC8HwGk0NCt	2026-05-18 21:16:05.419367591 +0000
@@ -1,3 +1,3 @@
-tests/json-test.toit:386:17: warning: Class 'BufferedReader' is deprecated. Use 'io.Reader' instead
+tests/json-test.toit:393:17: warning: Class 'BufferedReader' is deprecated. Use 'io.Reader' instead
     buffered := BufferedReader (TestReader it)
                 ^~~~~~~~~~~~~~
CMake Error at tests/tools/health_run.cmake:86 (message):
  Not equal
```

The easiest is to just run `make update-health-gold`

--- a/tests/health/gold/sdk/tests__json-test.toit.gold
+++ b/tests/health/gold/sdk/tests__json-test.toit.gold
@@ -1,3 +1,3 @@
-tests/json-test.toit:386:17: warning: Class 'BufferedReader' is deprecated. Use 'io.Reader' instead
+tests/json-test.toit:393:17: warning: Class 'BufferedReader' is deprecated. Use 'io.Reader' instead
     buffered := BufferedReader (TestReader it)
                 ^~~~~~~~~~~~~~

